components/places/src/import/ios/bookmarks.rs

```diff
@@ -176,12 +176,12 @@ fn populate_mirror_tags(db: &crate::PlacesDb) -> Result<()> {
     {
         let mut stmt = db.prepare(
             "SELECT mirror.id, stage.tags
-             FROM main.moz_bookmarks_synced mirror
-             JOIN temp.iosBookmarksStaging stage USING(guid)
-             -- iOS tags are JSON arrays of strings (or null).
-             -- Both [] and null are allowed for 'no tags'
-             WHERE stage.tags IS NOT NULL
-               AND stage.tags != '[]'",
+              FROM main.moz_bookmarks_synced mirror
```

Comment:
The whitespace changes here and below also look accidental? (If they aren't, I don't think they are adding enough value tbh, as it's not immediately clear if you are actually changing anything - I think not, but didn't read carefully enough to know for sure)

Reply:
Heh I didn't touch those queries, and in fact I could swear that `cargo fmt` introduced those changes once I changed something else in the file 👀